x-pack/test/functional/apps/uptime/locations.ts

```diff
@@ -28,21 +30,26 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         }
         return d;
       };
-      await makeChecksWithStatus(
-        getService('legacyEs'),
-        MONITOR_ID,
-        5,
-        2,
-        10000,
-        {},
-        'up',
-        mogrifyNoLocation
-      );
+      const esService = getService('legacyEs');
+      await makeChecksWithStatus(esService, MONITOR_ID, 5, 2, 10000, {}, 'up', mogrifyNoLocation);
+      await makeChecksWithStatus(esService, 'filter-testing', 50, 1, 10000, {}, 'up', (d: any) => {
+        d.observer.geo.name = 'filter-test';
```

Comment:
Maybe specify the geo name with a variable so we don't have to repeat this string value below.

Reply:
Addressed in https://github.com/elastic/kibana/pull/56794/commits/0a3f970f616280515ae3eee39c1fe49e86db1f15.